UnityMcpBridge/UnityMcpServer~/src/tools/manage_script_edits.py

```
@@ -0,0 +1,827 @@
+            text = text[:idx] + insert_text + text[idx:]
+        elif op == "replace_range":
+            start_line = int(edit.get("startLine", 1))
+            start_col  = int(edit.get("startCol", 1))
+            end_line   = int(edit.get("endLine", start_line))
+            end_col    = int(edit.get("endCol", 1))
+            replacement = edit.get("text", "")
+            lines = text.splitlines(keepends=True)
+            max_line = len(lines) + 1  # 1-based, exclusive end
+            if (start_line < 1 or end_line < start_line or end_line > max_line
+                    or start_col < 1 or end_col < 1):
+                raise RuntimeError("replace_range out of bounds")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)